docs/app-debug/debug-options.rst

```diff
@@ -0,0 +1,72 @@
+itself that are designed to help with debugging.
+These parameters :ref:`can be set <label-running-setting-mca-param-values>`
+in the usual ways.  MPI-level MCA parameters can be
+displayed by invoking the following command:
+
+.. code-block:: sh
+
+   # Use "--level 9" to see all the MCA parameters
+   # (the default is "--level 1"):
+   shell$ ompi_info --param mpi all --level 9
+
+Here is a summary of the debugging parameters for the MPI layer:
+
+* ``mpi_param_check``: If set to true (any positive value), and when
+  Open MPI is compiled with parameter checking enabled (the default),
+  the parameters to each MPI function can be passed through a series
+  of correctness checks.  Problems such as passing illegal values
+  (e.g., NULL or ``MPI_DATATYPE_NULL`` or other "bad" values) will be
+  discovered at run time and an MPI exception will be invoked (the
+  default of which is to print a short message and abort the entire
+  MPI job).  If set to false, these checks are disabled, slightly
+  increasing performance.
+
+* ``mpi_show_handle_leaks``: If set to true (any positive value),
+  Open MPI will display lists of any MPI handles that were not freed before
+  ``MPI_FINALIZE`` (e.g., communicators, datatypes, requests, etc.)
+
+* ``mpi_no_free_handles``: If set to true (any positive value), do not
+  actually free MPI objects when their corresponding MPI "free"
+  function is invoked (e.g., do not free communicators when
+  ``MPI_COMM_FREE`` is invoked).  This can be helpful in tracking down
+  applications that accidentally continue to use MPI handles after
+  they have been freed.
+
+* ``mpi_show_mca_params``: If set to true (any positive value), show a
+  list of all MCA parameters and their values during ``MPI_INIT``.
+  This can be quite helpful for reproducibility of MPI applications.
+
+* ``mpi_show_mca_params_file``: If set to a non-empty value, and if
+  the value of ``mpi_show_mca_params`` is true, then output the list
+  of MCA parameters to the filename value.  If this parameter is an
+  empty value, the list is sent to ``stderr``.
+
+* ``mpi_keep_peer_hostnames``: If set to a true value (any positive
+  value), send the list of all hostnames involved in the MPI job to
+  every process in the job.  This can help the specificity of error
+  messages that Open MPI emits if a problem occurs (i.e., Open MPI can
+  display the name of the peer host that it was trying to communicate
+  with), but it can somewhat slow down the startup of large-scale MPI
+  jobs.
+
+* ``mpi_abort_delay``: If nonzero, print out an identifying message
+  when ``MPI_ABORT`` is invoked showing the hostname and PID of the
+  process that invoked ``MPI_ABORT``, and then delay that many seconds
+  before exiting.  A negative value means to delay indefinitely.  This
+  allows a user to manually come in and attach a debugger when an
+  error occurs.  Remember that the default MPI error handler |mdash|
+  ``MPI_ERRORS_ABORT`` |mdash| invokes ``MPI_ABORT``, so this
```

Comment:
Swap the `MPI_ABORT` to
```
:ref:`MPI_Abort(3) <mpi_abort>`
```

Same with some other references to `MPI_ABORT` later in the file.